.circleci/config.yml

```diff
@@ -443,6 +443,42 @@ jobs:
       - store_artifacts: *artifacts_solc
       - persist_to_workspace: *artifacts_executables
 
+  t_osx_soltest: &t_osx_soltest
+    macos:
+      xcode: "11.0.0"
+    environment:
+      EVM: constantinople
+      OPTIMIZE: 0
+      TERM: xterm
+      CMAKE_PROGRAM_PATH: /Users/distiller/project/z3/bin
+      CMAKE_LIBRARY_PATH: /Users/distiller/project/z3/bin
+      CMAKE_INCLUDE_PATH: /Users/distiller/project/z3/include
+    steps:
+      - checkout
+      - attach_workspace:
+          at: build
+      - run:
+          name: Install dependencies
+          command: |
+            brew unlink python
+            brew install boost
+            brew install cmake
+            brew install wget
+            ./scripts/install_obsolete_jsoncpp_1_7_4.sh
+            brew install coreutils
+            wget https://github.com/Z3Prover/z3/releases/download/Z3-4.8.5/z3-4.8.5-x64-osx-10.14.2.zip
+            unzip z3-4.8.5-x64-osx-10.14.2.zip
+            rm -f z3-4.8.5-x64-osx-10.14.2.zip
+            mv z3-4.8.5-x64-osx-10.14.2 z3
```

Comment:
Could we install this to ``/usr/local`` and get rid of all the environment variable hacks way?

Reply:
I tried this first, but it looked like that the user doesn't have sufficient rights

Reply:
Yeah, ok, I kind of expected that.